ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1249,6 +1249,24 @@ FCU_CHWZTC2X_HT2XZTC2X_DFVSC2X:
   - CHWZTC2X
   - HT2XZTC2X
   - DFVSC2X
+  
+FCU_DFVSMC_CHWBRC_STM_RTM_FDPM_ZTC_SRC:
+  description: "FCU with Status, zone control, schedule run"
+  is_canonical: false
+  implements:
+  - FCU
+  - DFVSMC
+  - CHWBRC
+  - STM
+  - RTM
+  - FDPM
+  - ZTC
```

Comment:
How does it do zone temperature control if it also does return air temperature control? Which temperature does the valve control to?

Reply:
Hi sir, I'm reviewing again the points list and it turns out I need RTM + STM (device has return and supply temp sensors, no control) and then for discharge it is ZTC (zone temp sensor and setpoint).

Reply:
Not sure about chilled water sir. It is Pre-Cooled Fan Coil Units (Ceiling Concealed Ducted Type) but here is where it is confusing me too: it has 2 point list for zones that is described as "Zone Air Temperature", "Zone Temperature Set Point Adjustment/Control". the 1st one is sensor, but the 2nd one is a combined setpoint and command. so, CHWZTC for me is not an option(?) as it has 3 points for zone. I'm not sure what to do here.

this is the full point list for Pre-Cooled Fan Coil Units (Ceiling Concealed Ducted Type)

```
Supply Air Temperature
Return Air Temperature
Filter Status
Unit Enable
Unit Status
Unit Fault
Zone Air Temperature
Zone Temperature Set Point Adjustment/Control
Time Schedule
```

Reply:
same comment as above. it sounds like this is CHW cooling but you dont have the data for the chilled water valve. If its acually DX then you would use the DX equivlent (DXZTC), but would still have missing compressor information. This is totally fine, it just needs to reflect what the equipment actually is, even if telemetry is missing.